src/VisualStudio/VisualBasic/Impl/ObjectBrowser/VisualBasicLibraryServiceFactory.vb

```diff
@@ -0,0 +1,18 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.Host
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.VisualStudio.LanguageServices.Implementation.Library
+
+Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.ObjectBrowser
+    <ExportLanguageServiceFactory(GetType(ILibraryService), LanguageNames.VisualBasic), [Shared]>
+    Friend Class VisualBasicLibraryServiceFactory
+        Implements ILanguageServiceFactory
+
+        Public Function CreateLanguageService(languageServices As HostLanguageServices) As ILanguageService Implements ILanguageServiceFactory.CreateLanguageService
```

Comment:
Ditto earlier, you can just use ExportLanguageService if you hate the factory pattern.